pkg/middleware/middleware.go

```diff
@@ -140,6 +140,24 @@ func Process(quay *v1.QuayRegistry, obj client.Object, skipres bool) (client.Obj
 			return nil, err
 		}
 
+		if olabels := v1.GetLabelsOverrideForComponent(quay, kind); olabels != nil {
+			if dep.Labels == nil {
+				dep.Labels = map[string]string{}
+			}
+			for key, value := range olabels {
+				dep.Labels[key] = value
```

Comment:
I think that if we allow any override we can have other problems. I would say that the labels "quay-component", "app" and "quay-operator/quayregistry" should never be overwritten by the user. Can we get these exception here ?

Reply:
Should be handled now